### PR TITLE
Rr branch

### PR DESCRIPTION
**Pull request by raphaelroques**
header mods
footer mods 

worked on the first api call

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
```
Comment (connietee22) [inline index.css:9]: nice!

Comment (kdlry) [reply index.css:9]: Perfect!

Comment (afacey) [reply index.css:9]: No conflicts 🎉

**Review by connietee22 — CHANGES_REQUESTED** [review]
I hate this!

**Comment by raphaelroques** [comment]
> I hate this!

what ?

File: src/Footer.js [hunk 2]
```diff
@@ -1,7 +1,13 @@
 import React from 'react';
 
 const Footer = () => {
-  return <></>;
+  return (
+    <>
```
Comment (kdlry) [inline src/Footer.js:5]: Test: Maybe we should have a div here?

Comment (raphaelroques) [reply src/Footer.js:5]: ok good idea

**Pull request merged**